API Tokens, Allow a User Delete an API Token

wzalazar opened the pull request:
This PR introduces

- Button remove API Token
- Update Frost Client to 0.1.4

fix #41

lautarodragan commented:
The modal looks beautiful and the wording is just right! Awesome job :rocket: 

Some UI feedback...
- The "remove" (token) button looks broken.
- The individual rows in the table seem to have increased in height, but not so the table itself, thus overflowing it for some reason.
- The "close modal" button has an odd background
- When I try to delete a token nothing happens. Checking the network calls, I see the API returned `Internal Server Error`. 

![frost-delete-token](https://user-images.githubusercontent.com/2279687/38711966-1e9f2d8e-3e9f-11e8-9abe-8218315f1e46.png)

![close-modal](https://user-images.githubusercontent.com/2279687/38712015-6ef7db14-3e9f-11e8-9b03-9eebfb12cf5e.png)

wzalazar commented:
I fixed the following issues

- [ ] The "remove" (token) button looks broken.
This issue occurs in Firefox, it's was fix

- [ ] The individual rows in the table seem to have increased in height, but not so the table itself, thus overflowing it for some reason.
Fixed

- [ ]  The "close modal" button has an odd background
This issue occurs in Firefox, it's was fix

- [ ] When I try to delete a token nothing happens. Checking the network calls, I see the API returned Internal Server Error.
I created an PR in Frost API, this needs released
